spacy_udpipe/udpipe.py

```diff
@@ -40,11 +42,50 @@ def nextSentence(self, sentence: Sentence, _: ProcessingError) -> bool:
             if re.match(r"\W", token):
                 # leave no space after previous token iff current token
                 # is non-alphanumeric (i.e. punctuation)
-                prev_word.misc = self.NO_SPACE
+                prev_word.misc = NO_SPACE
             prev_word = word
         return True
 
 
+class Model(_Model):
+    """Model wrapper with pickling support, enabling multiprocess execution."""
+
+    def __init__(self, path: str):
+        self.path = path
+        self._model = super().load(path)
+        if self._model is None:
+            raise Exception(f"Cannot load UDPipe model from file '{path}'")
+
+    def __reduce__(self):
+        # pickle support
+        return (self.__class__, (self.path,))
+
+    @property
```

Comment:
The `@property` decorator is used as all of the `_Model` attributes are read-only.